recipes/perl-gd/meta.yaml

```diff
@@ -14,15 +14,13 @@ requirements:
     - llvm # [osx]
     - perl-threaded >=5.22.0
     - libgd >=2.2.3
-    - fontconfig # for libfontcontig.so
```

Comment:
Martin these `#` are not comments.
`#` is part of the template language. So maybe these things are interpreted as some variable?